de.wger.flutter.yml

```
@@ -0,0 +1,33 @@
+app-id: de.wger.flutter
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: wger
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+modules:
+  - name: wger
+    buildsystem: simple
+    build-commands:
+      - cp -R wger/bin/ /app/wger
+      - chmod +x /app/wger/wger
+      - mkdir /app/bin/
+      - mkdir /app/lib/
+      - ln -s /app/wger/wger /app/bin/wger
+      - install -Dm644 wger/icons/64x64/de.wger.flutter.png /app/share/icons/hicolor/64x64/apps/de.wger.flutter.png
+      - install -Dm644 wger/icons/128x126/de.wger.flutter.png /app/share/icons/hicolor/128x126/apps/de.wger.flutter.png
+      - install -Dm644 wger/icons/512x512/de.wger.flutter.png /app/share/icons/hicolor/512x512/apps/de.wger.flutter.png
+      - install -Dm644 wger/de.wger.flutter.desktop /app/share/applications/de.wger.flutter.desktop
+      - install -Dm644 wger/de.wger.flutter.appdata.xml /app/share/applications/de.wger.flutter.appdata.xml
```

```suggestion
      - install -Dm644 wger/de.wger.flutter.appdata.xml /app/share/appdata/de.wger.flutter.appdata.xml
```

that's why it can't find it.

---

yes, you are right